packages/components/src/components/dropdown-select/dropdown-select.tsx

```diff
@@ -233,11 +233,15 @@ export class DropdownSelect {
   }
 
   connectedCallback() {
+    if (this.hostElement.parentElement.tagName === 'SCALE-TAB-PANEL') {
```

Comment:
This would only work if the dropdown is a direct child of the tab panel. However, the issue also occurs when the dropdown is nested somewhere within the tab panel content.

What about this way of checking whether the dropdown is inside a tab panel:

```suggestion
    if (this.hostElement.closest("scale-tab-panel")) {
```

Reply:
I updated the code per your suggestion, but I will still ask about this in the team...honestly i would prefer to remove unnecessary code (if it actually is unnecessary) rather than adding another conditional 😅